### PR TITLE
Integration test/creating financial trasnaction

### PR DESCRIPTION
**Pull request by zetxdev**
integration test for creating Financial Transaction #180

### CODE REVIEW

**Review by dawciobiel — APPROVED** [review]
Looks good to me

File: src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/CreateFinancialTransactionIT.java [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+class CreateFinancialTransactionIT extends BaseIntegrationTestIT {
+
+    @Autowired
+    FinancialTransactionRepository financialTransactionRepository;
+
+    @Autowired
+    WalletRepository walletRepository;
+
+    private static BigDecimal TRANSACTION_AMOUNT_LIMIT = new BigDecimal("999999999999999.99");
```
Comment (dawciobiel) [inline src/test/java/pl/byczazagroda/trackexpensesappbackend/integration/CreateFinancialTransactionIT.java:32]:
```suggestion
   /**
   The maximum value for the `amount` parameter is described as:
   {@link  pl.byczazagroda.trackexpensesappbackend.dto.FinancialTransactionCreateDTO  FinancialTransactionCreateDTO}
   */
    private static final BigDecimal MAX_ALLOWED_TRANSACTION_AMOUNT = new BigDecimal("10000000000000.00");
```

**Review by dawciobiel — CHANGES_REQUESTED** [review]
Request changes

**Review by dawciobiel — APPROVED** [review]
Looks good for me.
GetAllWalletsIT - this file should be already on origin `develop`.

**Review by odzio33 — APPROVED** [review]
Looks good for me

**Pull request merged**